packages/wallets/src/features/cfd/screens/CompareAccounts/CompareAccountsCard.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { WalletText } from '../../../../components';
+import { THooks, TPlatforms } from '../../../../types';
+import { CFD_PLATFORMS } from '../../constants';
+import CompareAccountsButton from './CompareAccountsButton';
+import CompareAccountsDescription from './CompareAccountsDescription';
+import CompareAccountsPlatformLabel from './CompareAccountsPlatformLabel';
+import CompareAccountsTitleIcon from './CompareAccountsTitleIcon';
+import InstrumentsLabelHighlighted from './InstrumentsLabelHighlighted';
+import './CompareAccountsCard.scss';
+
+type TCompareAccountsCard = {
+    isAccountAdded: boolean;
+    isDemo: boolean;
+    isEuRegion: boolean;
+    isEuUser: boolean;
+    marketType: THooks.AvailableMT5Accounts['market_type'];
+    platform: TPlatforms.All;
+    shortCode: THooks.AvailableMT5Accounts['shortcode'];
+};
+
+const CompareAccountsCard = ({
+    isAccountAdded,
+    isDemo,
+    isEuRegion,
+    isEuUser,
+    marketType,
+    platform,
+    shortCode,
+}: TCompareAccountsCard) => {
+    return (
+        <div className='wallets-compare-accounts-main-container'>
+            <div className='wallets-compare-accounts-card-container'>
+                <CompareAccountsPlatformLabel platform={platform} />
+                {platform === CFD_PLATFORMS.CTRADER && (
+                    <div className='compare-cfd-account-card-container__banner'>
```

Comment:
hmm, that does not look consistent with BEM of this component, probably it shoudl be a block within wallets-compare-accounts-card

Reply:
Updated.